src/formatters/html.js

```diff
@@ -20,6 +20,10 @@ function htmlEscape(text) {
   return html;
 }
 
+HtmlFormatter.prototype.prepareContext = function(context) {
+  BaseFormatter.prototype.prepareContext.call(this, context);
+};
```

Comment:
So what happened in our Redux environment was that `context` was not loaded and populated with React state information. After this fix, it worked fine. IMO, this is required with JS's prototypical inheritance.